Modules/_testmultiphase.c

```diff
@@ -909,3 +909,20 @@ PyInit__test_non_isolated(void)
 {
     return PyModuleDef_Init(&non_isolated_def);
 }
+
+
+static PyModuleDef_Slot shared_gil_only_slots[] = {
+    {Py_mod_exec, execfunc},
+    {Py_mod_multiple_interpreters, Py_MOD_MULTIPLE_INTERPRETERS_SUPPORTED},
```

Comment:
Technically not needed, since `Py_MOD_MULTIPLE_INTERPRETERS_SUPPORTED` is the default.

```suggestion
```

Reply:
Yeah, I wanted to be explicit about it, to draw attention.  (Apparently it worked. 😄)  I'll add a note to that effect.

Reply:
Yeah, I actually wrote a "OTOH" sentence with that in mind, but deleted it before posting 😄 For tests, being explicit makes sense.